src/notes.tsx

```diff
@@ -36,6 +36,9 @@ import notesHistory from "notes/history";
 import hotkeys, { Hotkey } from "notes/hotkeys";
 import { sendMessage } from "messages";
 
+const getActiveFromUrl = (): string => window.location.search.startsWith("?") ? decodeURIComponent(window.location.search.substring(1)) : ""; // Bookmark
```

Comment:
Opening note as defined by the URL has the highest priority in determining which note to open.